Vision/vision2016.py

```diff
@@ -39,13 +39,13 @@ def findDistanceToTarget(width):
     return distance
 
 def findAngle(distance):
-    angle = math.degrees(math.atan(7/distance))
+    angle = (.1183*(distance **2 ) - (3.468 * distance) + 69.203)
```

Comment:
Ideally leave a comment here explaining that the values came from testing, or what they mean, etc.